HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -110,6 +109,193 @@ def test_ducts
+    _test_default_pv(hpxml, expected_interver_efficiency, expected_system_loss_frac)
+
+    default_hpxml('base-pv.xml')
+    model, hpxml = _test_measure(@args_hash)
+    expected_interver_efficiency = [0.96, 0.96]
+    expected_system_loss_frac = [0.14, 0.14]
+    _test_default_pv(hpxml, expected_interver_efficiency, expected_system_loss_frac)
+
+    hpxml = default_hpxml('base-pv.xml')
+    hpxml.pv_systems.each do |pv|
+      pv.year_modules_manufactured = 2010
+    end
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    expected_interver_efficiency = [0.96, 0.96]
+    expected_system_loss_frac = [0.182, 0.182]
+    _test_default_pv(hpxml, expected_interver_efficiency, expected_system_loss_frac)
+  end
+
+  def test_conditioned_building_volume
+    hpxml_name = 'base.xml'
+    hpxml = HPXML.new(hpxml_path: File.join(@root_path, 'workflow', 'sample_files', hpxml_name))
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    _test_default_conditioned_building_volume(hpxml, 21600)
+
+    default_hpxml('base.xml')
+    model, hpxml = _test_measure(@args_hash)
+    _test_default_conditioned_building_volume(hpxml, 27000)
+  end
+
+  def test_appliances
+    hpxml_name = 'base.xml'
+    hpxml = HPXML.new(hpxml_path: File.join(@root_path, 'workflow', 'sample_files', hpxml_name))
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    _test_default_clothes_washer(hpxml, [HPXML::LocationLivingSpace, 1.21, 380.0, 0.12, 1.09, 27.0, 3.2, 6.0, 1.0])
+    _test_default_clothes_dryer(hpxml, [HPXML::LocationLivingSpace, HPXML::ClothesDryerControlTypeTimer, 3.73, 1.0])
+    _test_default_dish_washer(hpxml, [HPXML::LocationLivingSpace, 307.0, 0.12, 1.09, 22.32, 4.0, 12, 1.0])
+    _test_default_refrigerator(hpxml, [HPXML::LocationLivingSpace, 650.0, 1.0])
+    _test_default_cooking_range(hpxml, [HPXML::LocationLivingSpace, false, 1.0])
+    _test_default_oven(hpxml, false)
+
+    hpxml = default_hpxml('base.xml')
+    hpxml.header.eri_calculation_version = '2014ADEGL'
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    _test_default_clothes_washer(hpxml, [HPXML::LocationLivingSpace, 0.331, 704.0, 0.08, 0.58, 23.0, 2.874, 6.0, 1.0])
+    _test_default_clothes_dryer(hpxml, [HPXML::LocationLivingSpace, HPXML::ClothesDryerControlTypeTimer, 2.62, 1.0])
+    _test_default_dish_washer(hpxml, [HPXML::LocationLivingSpace, 467.0, 0.12, 1.09, 33.12, 4.0, 12, 1.0])
+    _test_default_refrigerator(hpxml, [HPXML::LocationLivingSpace, 691.0, 1.0])
+    _test_default_cooking_range(hpxml, [HPXML::LocationLivingSpace, false, 1.0])
+    _test_default_oven(hpxml, false)
+
+    hpxml = default_hpxml('base.xml')
+    hpxml.header.eri_calculation_version = '2014ADEGL'
+    hpxml.clothes_dryers[0].fuel_type = HPXML::FuelTypeNaturalGas
```

Comment:
If I'm understanding correctly, you're trying to test a gas clothes dryer specifically in this test? Is that right? If so:
1) Can you put comments above each section that say what the test is for? E.g. # Test gas clothes dryer
2) No need to check values for refrigerator, cooking range, etc. Just test the things that will change.